rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -571,7 +593,7 @@ public void leaveLRA(URL lraId, String compensatorUrl) throws GenericLRAExceptio
 
             if (Response.Status.OK.getStatusCode() != response.getStatus()) {
                 LRALogger.i18NLogger.error_lraLeaveUnexpectedStatus(response.getStatus(), response);
-                throw new GenericLRAException(lraId, response.getStatus(), "", null);
+                throw new GenericLRAException(compensatorUrl, response.getStatus(), "", null);
```

Comment:
the exception message would be reasonable I think

Reply:
Same comment as before, so I will just raise a separate JIRA.

Reply:
Can we get the messages from LRALogger.i18NLogger.error_lraLeaveUnexpectedStatus(response.getStatus(), response);?

Reply:
We could but that requires extra analysis that is not called for in this JIRA (and I raised JBTM-3089 to add the extra text).

Reply:
I see, it was just a parameter name declaration change now